modal/_utils/grpc_utils.py

```diff
@@ -77,15 +77,32 @@ class RetryWarningMessage:
     errors_to_warn_for: typing.List[Status]
 
 
+class ConnectionPool:
```

Comment:
nit: This name could be a bit confusing since this isn't a connection pool in the traditional sense of an interchangeable pool of connections.

Reply:
good point. I updated to `ConnectionManager`